src/verify/VerificationCodeInputWrapper.test.tsx

```diff
@@ -0,0 +1,103 @@
+import { act, fireEvent, render, waitFor } from '@testing-library/react-native'
+import React from 'react'
+import SmsRetriever from 'react-native-sms-retriever'
+import VerificationCodeInputWrapper from 'src/verify/VerificationCodeInputWrapper'
+import { PhoneNumberVerificationStatus } from 'src/verify/hooks'
+import { mocked } from 'ts-jest/utils'
+
+const mockedSmsRetriever = mocked(SmsRetriever)
+
+describe('VerificationCodeInputWrapper', () => {
+  const mockOnSuccess = jest.fn()
+  const mockOnResendSms = jest.fn()
+  const mockSetSmsCode = jest.fn()
+  const mockPhoneNumber = '+15555555555'
+
+  const renderComponent = (
+    verificationStatus = PhoneNumberVerificationStatus.NONE,
+    showResend = true
+  ) =>
+    render(
+      <VerificationCodeInputWrapper
+        phoneNumber={mockPhoneNumber}
+        verificationStatus={verificationStatus}
+        onSuccess={mockOnSuccess}
+        onResendSms={showResend ? mockOnResendSms : undefined}
+        setSmsCode={mockSetSmsCode}
+      />
+    )
+
+  beforeEach(() => {
```

Comment:
Can we add a test to ensure the paste button appears if the clipboard contents are set appropriately (and that it doesn't appear if the contents are not valid)? We should be able to check the testID from [here](https://github.com/valora-inc/wallet/blob/main/src/components/ClipboardAwarePasteButton.tsx#L40).